src/libvirt-xml-parse.js

```diff
@@ -427,6 +428,17 @@ export function parseDumpxmlForDisks(devicesElem) {
                 readonly: !!readonlyElem,
                 shareable: !!shareableElem,
                 removable: targetElem.getAttribute('removable'),
+                address: { // optional
+                    type: addressElem ? driverElem.getAttribute('type') : undefined,
```

Comment:
Repeating the `addressElem` here is a bit nasty -- how about `address: addresElem ? { bus: ... } : {}`